azext_iot/constants.py

```diff
@@ -23,8 +23,8 @@
     "iothub-expiry",
     "iothub-deliverycount",
     "iothub-enqueuedtime",
-    "ContentType",
-    "ContentEncoding",
+    "content-type",
+    "content-encoding",
 ]
```

Comment:
We should update EVENT_LIB below (or refactor its use) so the out of band install, installs uamqp 1.6.6 (or whatever is appropriate) or higher